app-frontend/src/app/components/colorComposites/colorSchemeDropdown/colorSchemeDropdown.html

```diff
@@ -7,9 +7,9 @@
           uib-dropdown-toggle>
       <div class="gradient-bar gradient-bkg"
           ng-style="$ctrl.colorSchemeService.colorsToBackground(
-                      $ctrl.reflectedState.scheme.colors,
+                      $ctrl.state.scheme.colors,
                       90,
-                      $ctrl.reflectedState.blending.bins,
+                      $ctrl.state.blending.bins,
```

Comment:
Not exactly at this line, but if it would be simple, I think adding a `title` attribute to this div with the color scheme label would be nice.